airbyte-lib/airbyte_lib/_executor.py

```diff
@@ -123,7 +130,9 @@ def _get_connector_path(self) -> Path:
     def _run_subprocess_and_raise_on_failure(self, args: list[str]) -> None:
         result = subprocess.run(args, check=False)
         if result.returncode != 0:
-            raise Exception(f"Install process exited with code {result.returncode}")
+            raise exc.AirbyteConnectorInstallationError from exc.AirbyteSubprocessFailedError(
+                exit_code=result.returncode
```

Comment:
should we add stdout/stderr and args as context here?

Reply:
I left it out to avoid risk - and with a focus on parity. But yes, definitely we can add in the future. 👍